### DISCUSSION TITLE
Temporarily unimplemented features

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#18] [Ideas]
Browser audio management, paging or volume control in separate workspaces; for example, workspace 1 is playing video, and workspace 2 is playing audio; but I temporarily need the volume of this page of 2 to be greater than 1, and increase the volume of 1 as much as possible Low is not completely inaudible.

**Comment by gz83** [comment]
@whxuezhe Thorium currently has no related functions, you can try to find out if there is an extension that can meet your needs.

**Comment by Alex313031** [comment]
@whxuezhe Yeah, you can change the flags chrome://flags/#global-media-controls-modern-ui chrome://flags/#tab-search-media-tabs to help, but it wont do all of what you're looking for. Like @gz83 said I would try finding an extension.

Thorium already has this flag > chrome://flags/#enable-tab-audio-muting forced to true in source code, so you can also quickly mute tabs with a single click.

**Reply by whxuezhe** [reply]
@Alex313031 I saw it in chrome canary, and the implementation details are not clear.

**Reply by Alex313031** [reply]
@whxuezhe Thorium is always built to tip-o-tree chromium, so whenever it lands in chrome canary, you will see it in thorium.

**Reply by whxuezhe** [reply]
@Alex313031  chrome://flags/#global-media-controls-modern-ui chrome://flags/#tab-search-media-tabs The operation is done, but it will become very stuck, or even suspended, and resume after closing.[https://cloud.189.cn/web/share?code=VzuIN37Vfemm（访问码：q9zj）](url)Want to achieve the effect of national cloud disk picture display

**Reply by gz83** [reply]
> @Alex313031 chrome://flags/#global-media-controls-modern-ui chrome://flags/#tab-search-media-tabs The operation is done, but it will become very stuck, or even suspended, and resume after closing.[https://cloud.189.cn/web/share?code=VzuIN37Vfemm（访问码：q9zj）](url)Want to achieve the effect of national cloud disk picture display

@whxuezhe You can directly post the picture in the reply, which is more convenient for us to view without uploading to the network disk.

The image below is what you need to upload.

![image](https://user-images.githubusercontent.com/17492865/177069634-25136b2e-af80-455d-8490-3a70c27abaae.png)

By the way, have you tried the AVX version of Thorium? Its link is below.

https://github.com/Alex313031/Thorium-Win/releases/tag/M105.0.5145.0

EDIT: Intel's 12th generation processor architecture design is different from the previous one. Some cores may not contain the AVX2 instruction set, so it is recommended that you use the regular AVX version, and it may solve the crash problem you encounter.
